include/openspace/documentation/documentationengine.h

```diff
@@ -109,6 +110,8 @@ class DocumentationEngine : public DocumentationGenerator {
 
     std::string generateJson() const override;
 
+    nlohmann::json generateJsonJson() const;
```

Comment:
The name still cracks me up

Reply:
I'll change it when I remove the regular generateJson functions:)